### DISCUSSION TITLE
Even if I disable the extension in 'EXAppExtensionBrowserViewController,' the extension still works.

### DISCUSSION THREAD

**Discussion by pookjw** [insidegui/TextTransformer#4] [Q&A] [answered]
In this code

https://github.com/insidegui/TextTransformer/blob/985efd0eb48650b996b1b28c3d3ced0c186929fd/TextTransformer/Extension%20Hosting/TextTransformExtensionHost.swift#L57

If I make this change like below, the extension will remain in `self.extensions` even if it is disabled in EXAppExtensionBrowserViewController.

```swift
if identities.count == 2 {
    self.extensions = identities.map { TextTransformExtensionInfo(with: $0) }
}
```

<img width="803" alt="Screenshot 2024-08-24 at 02 04 58" src="https://github.com/user-attachments/assets/71ff17d6-3522-4692-8322-cc2c087e8b2c">

As you can see, Uppercase extension is disabled. If I select Uppercase extension and try to transform, it still works even if Uppercase extension is disabled.

Is this a bug in ExtensionFoundation?

---

If I try Objective-C API of `ExtensionFoundation` (yes, it's Private) [Example](https://github.com/pookjw/CoreDataXPCStore)

`-[_EXExtensionRecordIdentity enabled]` is `NO`, however, there is the same issue as with the Swift API.

**Comment by insidegui** [comment, marked as answer]
I think extensions embedded in the same app that's running the extension are always enabled. The enabled/disabled state only applies to other apps
